child-template/README.md

```diff
@@ -0,0 +1,137 @@
+
+### Baseline Deployment
+
+1. Navigate to the `child-template/examples` folder.
+2. Copy the `baseline_terraform.tfvars.template` file to the root of the `child-template` folder.
+3. Rename the file to `baseline_terraform.tfvars`.
+4. Customize the `baseline_terraform.tfvars` file with your environment-specific variables. See [Baseline Deployment Variables](#baseline-deployment-variables) for more details about the variables. 
+5. Run the following commands from the root of `child-template` folder:
+
+    ```bash
+    terraform plan -var-file="baseline_terraform.tfvars"
+    terraform apply -var-file="baseline_terraform.tfvars"
+    ```
+
+6. When prompted, confirm the changes by typing "yes" and pressing Enter.
+7. Ensure that the deployment successfully completes.
+
+#### Baseline Deployment Variables
+
+Following variables can be customized to suit the deployment of your baseline child deployment: 
+
+| Name | Description | Default Value |
+|------|-------------|---------------|
+| resource_label | Unique label to identify the resources created as part of the Landing Zone | "" |
+| enable_domain_replication | Enables the replication of the identity domain | true |
+| identity_domain_license_type | Type of license used for the identity domain. | "premium" |
+| realm_key | Define what realm you are going to deploy the LZ into. Default is OC3 | "3" |
+| home_region_deployment | Deploy the LZ in your home region? | true |
+| enable_logging_compartment | Enable the creation of a logging compartment  | true |
+| central_vault_type | Use a central vault | "DEFAULT" |
+| enable_vault_replica | Use vault replica | false |
+| enable_cloud_guard | Enable Cloud Guard. This should only be set to true when your tenancy has not yet enabled Cloud Guard. | true |
+| bastion_client_cidr_block_allow_list | List of CIDR ranges that will be granted access to the bastion | ["15.0.0.0/16"] |
+| enable_bastion | Enable the bastion service | true |
+| vdms_critical_topic_endpoints | List of email address who will receive the VDMS critical topics | [] |
+| vdms_warning_topic_endpoints | List of email address who will receive the VDMS warning topics | [] |
+| vdss_critical_topic_endpoints | List of email address who will receive the VDSS critical topics | [] |
+| vdss_warning_topic_endpoints | List of email address who will receive the VDSS warning topics | [] |
+| enable_vdss_warning_alarm | Enable the VDSS warning alarm | true |
+| enable_vdss_critical_alarm | Enable the VDSS critical alarms | true |
+| enable_vdms_warning_alarm | Enable the VDMS warning alarms | true |
+| enable_vdms_critical_alarm | Enable the VDMS critical alarms | true |
+| onboard_log_analytics | Onboard on the log analytics service | false |
+| vdss_vcn_cidr_block | CIDR block used for the VDSS VCN | "15.1.0.0/16" |
+| lb_subnet_cidr_block | Load balancer subnet CIDR range | "15.1.1.0/24" |
+| lb_subnet_name | Name used for the Load Balancer subnet | "OCI-SCCA-PARENT-LZ-VDSS-LB-SUBNET" |
```

Comment:
Lines 65, 67, and 73 should be named after OCI-SCCA-CHILD-LZ not OCI-SCCA-PARENT-LZ